porch/repository/pkg/git/git_test.go

```diff
@@ -568,3 +568,62 @@ func TestListPackagesDrafts(t *testing.T) {
 		t.Errorf("Package Revisions in drafts-repository: (-want,+got): %s", cmp.Diff(want, got))
 	}
 }
+
+func TestApproveDraft(t *testing.T) {
+	tempdir := t.TempDir()
+	tarfile := filepath.Join("testdata", "drafts-repository.tar")
+	repo, address := ServeGitRepository(t, tarfile, tempdir)
+
+	const (
+		repositoryName                            = "approve"
+		namespace                                 = "default"
+		draftReferenceName plumbing.ReferenceName = "refs/heads/drafts/bucket/v1"
```

Comment:
I wonder if we should think about concurrent drafts i.e. what if you and I both want to think about changing a package?